lib/flipper/utilities/bulk_feature_checker.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module Flipper
+  module Utilities
+    module BulkFeatureChecker
+      # Accepts an array of feature keys (symbol or string format) and returns a result hash of which keys
+      # are enabled, disabled, or missing; feature keys are returned in symbol format
+      def self.status(features)
```

Comment:
I think renaming this to `enabled_status` is a bit clearer to the intent, since Flipper allows for partial-rollouts which this method doesn't concern itself with.

Reply:
Good call, I've made that change. Thanks!